index.js

```diff
@@ -28,7 +28,7 @@ function forwarded (req) {
 
   // simple header parsing
   var proxyAddrs = parse(req.headers['x-forwarded-for'] || '')
-  var socketAddr = req.connection.remoteAddress
+  var socketAddr = req.socket.remoteAddress
```

Comment:
```suggestion
  var socketAddr = req.socket ? req.socket.remoteAddress : req.connection.remoteAddress
```

A test should be added to cover this case.